src/crate/theme/rtd/crate/footer.html

```diff
@@ -73,33 +73,3 @@ <h4 class="c-footer-heading">
     </div>
   </div>
 </div>
-
-
-<!--
-<footer class="footer">
-  <div class="footer-container">
-    <p class="pull-right">
-      <a href="#">Back to top</a>
-      {% if theme_source_link_position == "footer" %}
-      <br/>
-      {% include "sourcelink.html" %}
-      {% endif %}
```

Comment:
would sourcelink allow us to link to the github page of the source doc? that would be really cool. by chance, do you know?